src/components/Carousel/Carousel.tsx

```diff
@@ -201,14 +246,16 @@ const Carousel: React.FC<CarouselProps> = ({
     const containerEle = containerRef.current as HTMLDivElement
     const carouselItems = containerEle.children
 
-    if (length < 2) {
-      setOneCarouselStyle(curIndex, carouselItems)
-    } else if (length < 3) {
-      setTwoCarouselStyle(length, curIndex, width, depth, carouselItems)
-    } else if (length < 5) {
-      setThreeCarouselStyle(length, curIndex, width, depth, carouselItems)
-    } else {
-      setFiveCarouselStyle(length, curIndex, width, depth, carouselItems)
+    if (!pause.current) {
```

Comment:
The `if (!pause.current) {}` condition can be removed because this useEffect function is triggered `by curIndex`. And the functions (SlidePrev and SlideNext) that change the `curIndex` are already prevented when `pause.current === true`.

Reply:
Done !